app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t(".app_name"), root_path(locale: I18n.locale), id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("static_pages.home.title") ,    root_path %></li>
+        <li><%= link_to t("static_pages.help.title") ,    help_path %></li>
+        <li><%= link_to t("static_pages.layouts.header.sign_in") , '#' %></li>
```

Comment:
Sai coding convention: 
- nháy đôi nào
- space sau dấu phẩy

=> self check toàn bộ html.erb